jsonapi_parameters.gemspec

```diff
@@ -5,8 +5,8 @@ require 'jsonapi_parameters'
 Gem::Specification.new do |spec|
   spec.name        = 'jsonapi_parameters'
   spec.version     = JsonApi::Parameters::VERSION
```

Comment:
as far as I see the version is not bumped but I guess we should do it

Reply:
I amended the commit but didn't force push, my bad. Good catch!